Day-II-Calculator/calculator/src/components/ButtonComponents/OperatorButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Button.css';
+
+const OperatorButton = () => {
+    return (
+        <div className="operators">
+            <div className="operator division">/</div>
+            <div className="operator multiplication">x</div>
+            <div className="operator subtraction">-</div>
+            <div className="operator addition">+</div>
+            <div className="operator equal">=</div>
+        </div>
```

Comment:
Same here, these should all be separate renders of an operator button component at a higher level in the program.